.rubocop_todo.yml

```diff
@@ -1,34 +1,47 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-20 14:00:00 -0700 using RuboCop version 0.34.2.
+# on 2015-10-27 17:33:47 -0700 using RuboCop version 0.34.2.
```

Comment:
I can rerun.  I think violations are going down.  This code also hasn't had autocorrect yet, so let's not optimize prematurely?